packages/web-forms/src/components/common/ValidationMessage.vue

```diff
@@ -1,9 +1,22 @@
 <script setup lang="ts">
-withDefaults(defineProps<{ message?: string; showMessage?: boolean; addPlaceholder?: boolean }>(), {
-	message: undefined,
-	showMessage: false,
-	addPlaceholder: true,
-});
+import { QUESTION_HAS_ERROR } from '@/lib/constants/injection-keys.ts';
+import { type ComputedRef, inject, computed } from 'vue';
+
+withDefaults(
+	defineProps<{
+		message?: string;
+		addPlaceholder?: boolean;
+	}>(),
+	{
+		message: undefined,
+		addPlaceholder: true,
+	}
+);
+
+const showMessage = inject<ComputedRef<boolean>>(
+	QUESTION_HAS_ERROR,
+	computed(() => false)
+);
```

Comment:
I'm not a big fan of Vue’s provide/inject because it feels like magic values with unclear origins. I replaced them with Symbols (QUESTION_HAS_ERROR, SUBMIT_PRESSED) in injection-keys.ts. This change improves things slightly by ensuring uniqueness, avoiding typos in strings, and it's recommended in [Vue's official documentation.](https://vuejs.org/guide/components/provide-inject#working-with-symbol-keys)